File: templates/matlab/chemistry_spectra_decision_boundary.m
function fig = chemistry_spectra_decision_boundary()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('decision_map', 1921, 'chemistry spectra: decision boundary', 'chemistry spectra', 'decision boundary');
end

File: templates/python/chemistry_spectra_decision_boundary.py
"""chemistry_spectra_decision_boundary: 化学谱图决策边界图（decision-map 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='chemistry spectra: decision boundary'):
    return make_template_figure('decision_map', seed=1921, title=title, domain='chemistry spectra', topic='decision boundary')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
